RELEASE.md

```diff
@@ -24,11 +24,13 @@ The next sections include detailed instructions on how to complete each step of
 
 ## 1. Deciding to release a new version
 
--   [ ] 1.  Inform the engineering team that you are the release manager for a new release in the Slack channel #team-engineering (tag @engineers) and get a status from each squad on any changes that may be on the master branch of components that may be deal breakers for the release.
+-   [ ] 1.  Inform the engineering team that you are the release manager for a new release in the Slack channel [#team-engineering](https://codacy.slack.com/channels/team-engineering) (tag @engineers) and get a status from each squad on any changes that may be on the master branch of components that may be deal breakers for the release.
 
--   [ ] 2.  Confirm with every squad if the release can go ahead.
+-   [ ] 2.  Inform all stakeholders outside the engineering team that a new release proccess is being started, in the Slack channel [#sh_releases](https://codacy.slack.com/channels/sh_releases) (tag @here).
 
--   [ ] 3.  Decide the new version number.
+-   [ ] 3.  Confirm with every squad if the release can go ahead.
```

Comment:
I think this step could be swapped with step 2., or maybe even better, it could be part of step 1. directly.